consensus/enclave/api/src/messages.rs

```diff
@@ -58,6 +58,11 @@ pub enum EnclaveCall {
     /// Retrieves the block signer (Ed25519 public key) of an enclave.
     GetSigner,
 
+    /// The [ConsensusEnclave::get_fee_recipient()] method.
+    ///
+    /// Retrieves the fee recipient (RistrettoPublic, RistrettoPublic) for the enclave.
+    GetFeeRecipient,
```

Comment:
Can't that be returned as a PublicAddress?

Reply:
We would need to add derive serde serialize to PublicAddress to make that go. I think we could do that, but I wanted to get the lowest hanging fruit first.

Reply:
I have to say, having to use `.0` and `.1` in the code below is pretty error-prone. If deriving serde on PublicAddress is easy then it might be a better solution (until we move away from serde completely for enclave stuff)

Reply:
Deriving it was straight forward, but it resulted in issues with no_std. I'm tracking those down, but will move forward without it for now.

Reply:
Went with @eranrund's suggestion to add FeePublicKey to consensus_enclave_api to make using the spend/view components less error prone.